### DISCUSSION TITLE
GoPro Hero 11 Black Mini support

### DISCUSSION THREAD

**Discussion by struffel** [gopro/OpenGoPro#236] [Q&A] [answered]
Looking at the compatibility information on the [HTTP Spec](https://gopro.github.io/OpenGoPro/http_2_0) I see support for the Hero 9/10/11 Black, but not for the upcoming Hero 11 *Mini*. 
Has it simply not been added to the documentation because it's not officially out yet or is API access generally not available for the "Mini" edition?

**Comment by KonradIT** [comment]
Going to venture a guess that HTTP over USB calls will not be possible due to lack of webcam functionality, the rest should be either identical (BLE) or very similar (wifi using gpControl aka Smarty calls)

**Comment by tcamise-gpsw** [comment, marked as answer]
@StruffelProductions that's correct, the specs will be updated when the camera is officially released

**Reply by NickPronto** [reply]
Any teasers on functionality that won’t be possible with the Mini? Hoping there are none (apart from steaming)! 🥰

**Comment by CharlesMod** [comment]
For the record, I can control my HERO 11 mini over USB with HTTP just fine

**Reply by KonradIT** [reply]
Can you init a live preview as well?
